tools/clang/test/HLSL/ShaderOpArith.xml

```diff
@@ -2966,6 +3030,36 @@ void main(uint TID: SV_GroupIndex) {
   Values[Idx].x = QuadAny(ThreadBool) ? 1 : 2;
   Values[Idx].y = QuadAll(ThreadBool) ? 3 : 4;
 }
+
+[numthreads(8, 8, 1)]
+void CSMain(uint GID: SV_GroupIndex) {
+  QuadAnyAllTest(GID, 0);
+}
+
+struct Payload {
+  uint nothing;
+};
+
+[numthreads(8, 8, 1)]
+void ASMain(uint GID : SV_GroupIndex) {
+  QuadAnyAllTest(GID, 0);
+  Payload P = {0};
```

Comment:
Please move payload to groupshared global.  We don't want to encourage this pattern in any of our code.  I think we'll want to prevent it in the future even.

Reply:
I see that this is a pattern throughout this file, so perhaps that can be cleaned up with another change...